rspace/src/main/scala/coop/rchain/rspace/history/InsertException.scala

```diff
@@ -0,0 +1,3 @@
+package coop.rchain.rspace.history
+
+class InsertException(msg: String) extends Exception(msg)
```

Comment:
WDYT about moving this along with `LookupException` to `exceptions.scala`? Less files to jump between

Reply:
Yep, makes sense to me.  Will do.